src/pages/p2p/components/_availability.tsx

```diff
@@ -183,14 +185,12 @@ const Availability = () => {
                                 <Localize
                                     translate_text="<0>Log in</0> to your Deriv account. Don’t have one? <1>Sign up</1> for free."
                                     components={[
-                                        <SpanLinkText
+                                        <StyledLinkText
                                             id="dm-p2p-login-link"
-                                            size={24}
                                             onClick={handleLogin}
                                             color="red"
-                                            external
                                             key={0}
-                                        />,
+                                        ></StyledLinkText>,
```

Comment:
Can this be a self closing tag too?

Reply:
hi @konstantinosG-deriv , made changes in new commit :)